tests/load-tests/run.sh

```diff
@@ -7,5 +7,5 @@ if [ -z ${DOCKER_CONFIG_JSON+x} ]; then
 else echo "DOCKER_CONFIG_JSON is set"; fi
 
 USER_PREFIX=${USER_PREFIX:-testuser}
-go run loadtest.go --username "$USER_PREFIX" --users "${USERS_PER_THREAD:-50}" -w -l -t "${THREADS:-1}" --disable-metrics &&
+go run loadtest.go --component-repo "$COMP_URL" --username "$USER_PREFIX" --users "${USERS_PER_THREAD:-50}" -w -l -t "${THREADS:-1}" --disable-metrics &&
```

Comment:
We should set a default value for the component repo and also please use self-explanatory env variables names.

```suggestion
go run loadtest.go --component-repo "${COMPONENT_REPO:-https://github.com/devfile-samples/devfile-sample-code-with-quarkus}" --username "$USER_PREFIX" --users "${USERS_PER_THREAD:-50}" -w -l -t "${THREADS:-1}" --disable-metrics &&
```

Reply:
Yeah, looks better and also provides default.

Reply:
@naftalysh Please incorporate the above change. Thx

Reply:
> We should set a default value for the component repo and also please use self-explanatory env variables names.

@naftalysh Please incorporate the above changes

Reply:
@pmacik 
Yes, I have incorporated the above changes
Please approve the the review changes